### PR TITLE
Selc 213

### PR DESCRIPTION
**Pull request by antonioT90**
### List of changes

upload of informativa privacy and t&c on checkout storage and served using CDN
(the real file will be replaced when provided)

### Type of changes

- [x] Add new resources
- [ ] Update configuration to existing resources
- [ ] Remove existing resources

### Env to apply

- [x] DEV
- [ ] UAT
- [ ] PROD

### Does this introduce a change to production resources with possible user impact?

- [ ] Yes, users may be impacted applying this change
- [x] No

### Does this introduce an unwanted change on infrastructure? Check terraform plan execution result

- [ ] Yes
- [x] No

### Other information

---

### CODE REVIEW

File: src/core/assets.tf [hunk 1]
```diff
@@ -0,0 +1,29 @@
+data "local_file" "tc" {
+  filename = "${path.module}/assets/tc.pdf"
+}
```
Comment (pasqualedevita) [inline src/core/assets.tf:3]: can we use extended name, ie terms-condition?

Comment (antonioT90) [reply src/core/assets.tf:3]: I'll change this approach using the same as contracts template

Comment (antonioT90) [reply src/core/assets.tf]: would you like prefer to rename also the file?

Comment (iwoak) [reply src/core/assets.tf]: @antonioT90 yes, could you change the file name to extended version like terms-condition?

Comment (antonioT90) [reply src/core/assets.tf]: renamed, InformativaPrivacy.pdf is ok?

Comment (antonioT90) [reply src/core/assets.tf]: renamed into privacy-disclaimer.pdf

**Comment by iwoak** [comment]
@pasqualedevita could you review this after last update from @antonioT90 ?

**Review by pasqualedevita — APPROVED** [review]

**Pull request merged**